pkg/activator/net/revision_backends.go

```diff
@@ -64,6 +64,10 @@ type revisionDestsUpdate struct {
 const (
 	probeTimeout   time.Duration = 300 * time.Millisecond
 	probeFrequency time.Duration = 200 * time.Millisecond
+
+	// UserAgent is the user-agent header value set in probe requests sent
+	// from activator to user pods.
+	UserAgent = "Knative-Activator-Probe"
```

Comment:
we have a generic place in `apis/networking/register.go` for headers, IIRC.

Reply:
`apis/networking/register.go` is for labels. You mean `pkg/network/network.go`?

Reply:
Sure :)

Reply:
Done